x/auth/types/querier.go

```diff
@@ -19,3 +19,13 @@ type QueryAccountParams struct {
 func NewQueryAccountParams(addr sdk.AccAddress) QueryAccountParams {
 	return QueryAccountParams{Address: addr}
 }
```

Comment:
We don't need `QueryAccountParams` anymore, replace it with `QueryAccountRequest`